Fhi.HelseId.Web/DPoP/DPoPContext.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace Fhi.HelseId.Web.DPoP;
+
+public class DPoPContext
+{
+    public const string ContextKey = "request_has_jkt_context";
```

Comment:
Er dette navnekonvensjonen?

Reply:
Det har ikke noe å si ettersom det kun er en verdi vi bruker internt i tjenesten for å identifisere om et HTTP-kall er gjort i kontekst av DPoP i HttpContexten